Python/Tags_Scraper/Requirements.txt

```diff
@@ -0,0 +1,3 @@
+Libraries which will be required and need to be installed are:
```

Comment:
You don't need to add this line in your `requirements.txt`

You can generate a Requirements File via `pip freeze` and remove all the unnecessary lines.